domain/schema/model.go

```diff
@@ -19,10 +19,12 @@ const (
 	tableVolume
 	tableVolumeAttachment
 	tableVolumeAttachmentPlan
-	tableSecretAutoPrune
+	tableSecretMetadata
```

Comment:
I think this should be "columnSecretAutoPrune" or something like that since it's a column trigger and not the whole table